### PR TITLE
API 5031 Open API Conformance Statement

### PR DESCRIPTION
**Pull request by that-guy-is-it**

### CODE REVIEW

File: smart-cards/pom.xml [hunk 1]
```diff
@@ -24,4 +25,32 @@
...
   </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>io.swagger.core.v3</groupId>
+        <artifactId>swagger-maven-plugin</artifactId>
+        <version>${swagger-maven-plugin.version}</version>
+        <executions>
+          <execution>
+            <id>r4-smart-cards-swag</id>
```
[the remark this answers is not in the document]
Comment (jblefkove-va) [reply smart-cards/pom.xml:36]: sorry, `r4-open-api`. When `dstu2` is added that will have its own id as well.

File: smart-cards/src/main/java/gov/va/api/health/smartcards/R4MetadataController.java [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+        .name(buildProperties.getGroup() + ":" + buildProperties.getArtifact())
+        .releaseDate(buildProperties.getTime().toString())
+        .version(buildProperties.getVersion())
+        .build();
+  }
+
+  @Getter
+  @AllArgsConstructor
+  enum SearchParam {
+    _ID("_id", CapabilityStatement.SearchParamType.token);
```
Comment (jblefkove-va) [inline smart-cards/src/main/java/gov/va/api/health/smartcards/R4MetadataController.java:109]: Hmm, for this app we don't actually support any searches...

Comment (that-guy-is-it) [reply smart-cards/src/main/java/gov/va/api/health/smartcards/R4MetadataController.java:109]: 🤦 is all I can say.

File: smart-cards/src/main/java/gov/va/api/health/smartcards/patient/SmartCardsPatientApi.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+import io.swagger.v3.oas.annotations.Parameter;
+import io.swagger.v3.oas.annotations.enums.ParameterIn;
+import io.swagger.v3.oas.annotations.parameters.RequestBody;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import org.springframework.http.ResponseEntity;
+
+public interface SmartCardsPatientApi {
+  @Operation(summary = "Issues Verfiable Credential")
```
Comment (apeirats-va) [inline smart-cards/src/main/java/gov/va/api/health/smartcards/patient/SmartCardsPatientApi.java:14]: typo

File: smart-cards/src/test/java/gov/va/api/health/smartcards/R4MetadataControllerTest.java [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+                .interaction(
+                    List.of(
+                        CapabilityStatement.ResourceInteraction.builder()
+                            .code(CapabilityStatement.TypeRestfulInteraction.read)
+                            .documentation(
+                                "Implemented per specification. See http://hl7.org/fhir/R4/http.html")
+                            .build(),
+                        CapabilityStatement.ResourceInteraction.builder()
+                            .code(CapabilityStatement.TypeRestfulInteraction.search_type)
+                            .documentation(
```
Comment (apeirats-va) [inline smart-cards/src/test/java/gov/va/api/health/smartcards/R4MetadataControllerTest.java:134]: no searches?

**Review by jblefkove-va — APPROVED** [review]

**Pull request merged**